meson.build

```diff
@@ -231,10 +231,29 @@ else
   lvfs_metadata_format = 'gz'
 endif
 conf.set_quoted('FU_LVFS_METADATA_FORMAT', lvfs_metadata_format)
-gusb = dependency('gusb', version: '>= 0.4.8', fallback: ['gusb', 'gusb_dep'], required: get_option('gusb'))
-if gusb.found()
-  conf.set('HAVE_GUSB', '1')
+
+# FreeBSD is missing some libusb symbols
+libusb = dependency('libusb-1.0', version : '>= 1.0.9', required: get_option('usb'))
```

Comment:
how about picking a more modern line in the sand to start like 1.0.25 or 1.0.27?

Reply:
It's a good idea -- we can bump to 0.1.12 and get rid of quite a few ifdefs. FreeBSD only supports an API compatible with 0.1.12 unfortunately.